### PR TITLE
Add new version of container crontab

### PR DESCRIPTION
**Pull request by galal-hussein**
https://github.com/rancher/rancher/issues/8979

### CODE REVIEW

**Comment by deniseschannon** [comment]
Can you separate out to 2 different commits? One for the base copy of the previous version and then one with the changes.

**Comment by galal-hussein** [comment]
@deniseschannon Done.

File: infra-templates/container-crontab/1/rancher-compose.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .catalog:
   name: "Rancher Container Crontab"
-  version: "v0.1.0-rancher1"
+  version: "v0.2.0-rancher1"
```
Comment (deniseschannon) [inline infra-templates/container-crontab/1/rancher-compose.yml]: can we just change the version to `v0.2.0`

File: infra-templates/container-crontab/1/rancher-compose.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
   description: "Service that will start, stop or restart containers"
   minimum_rancher_version: v1.4.0
```
Comment (deniseschannon) [inline infra-templates/container-crontab/1/rancher-compose.yml]: does this work in all versions or only certain versions?

also, can you add `maximum_rancher_version` to the template in folder `0`

**Pull request merged**